build.gradle

```diff
@@ -123,6 +123,16 @@ dependencyManagement {
     dependencySet(group: 'com.google.guava', version: '29.0-jre') {
       entry 'guava'
     }
+    dependencySet(group: 'org.springframework', version: '5.2.5.RELEASE') {
+      entry 'spring-core'
+      entry 'spring-context'
+      entry 'spring-expression'
+      entry 'spring-beans'
+      entry 'spring-web'
```

Comment:
aye, ccd client came in :(. can we exclude any health check autoconfig from there? probably non of `web` is requirec either

Reply:
I disbaled all health checks. I will try to exclude jar when I am finished with all ccd actions.